internal/repositories/address.go

```diff
@@ -0,0 +1,39 @@
+package repositories
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/thirdfort/thirdfort-go-code-review/internal/models"
+)
+
+func (s *Store) FindAddress(ctx context.Context, transactionId string) (*models.Address, error) {
+	txn := models.Transaction{}
+	s.db.Where(fmt.Sprintf("id = %v", transactionId)).First(&txn)
+
+	if txn.Address == nil {
+		return nil, errors.New("no transaction")
+	} else {
+		return txn.Address, nil
+	}
+}
+
+func (s *Store) UpdateAddress(ctx context.Context, transactionID string, newAddress models.Address) (*models.Address, error) {
```

Comment:
I can't see the transitions for the status column e.g. where the status is updated to "completed" after successful update operation?